website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -425,13 +436,13 @@ The following variables should be present in the `cumulus-tf/orca_variables.tf`
 file. The variables must be set with proper values for your environment in the
 `cumulus-tf/terraform.tfvars` file.
 
-| Variable               | Definition                                              | Example Value                 |
-| ---------------------- | --------------------------------------------------------| ----------------------------- |
-| `db_admin_password`    | Password for RDS database administrator authentication  | "My_Sup3rS3cr3t_admin_Passw0rd"|
-| `db_host_endpoint`     | Database host endpoint to connect to.                   | "aws.postgresrds.host"        |
-| `db_user_password`     | Password for RDS database user authentication           | "My_Sup3rS3cr3tuserPassw0rd"  |
-| `orca_default_bucket`  | Default ORCA S3 Glacier bucket to use.                  | "PREFIX-orca-primary"         |
-
+| Variable                | Definition                                              | Example Value                 |
+| ----------------------  | --------------------------------------------------------| ----------------------------- |
+| `db_admin_password`     | Password for RDS database administrator authentication  | "My_Sup3rS3cr3t_admin_Passw0rd"|
+| `db_host_endpoint`      | Database host endpoint to connect to.                   | "aws.postgresrds.host"        |
+| `db_user_password`      | Password for RDS database user authentication           | "My_Sup3rS3cr3tuserPassw0rd"  |
+| `orca_default_bucket`   | Default ORCA S3 Glacier bucket to use.                  | "PREFIX-orca-primary"         |
+| `rds_security_group_id` | Cumulus' RDS Security Group's ID.                       | "g-01234567890123456"         |
```

Comment:
example value missing a `s` in the beginning

Reply:
Fixed